packages/driver/cypress/e2e/e2e/origin/origin.cy.ts

```diff
@@ -166,7 +166,6 @@ describe('cy.origin', () => {
         cy.on('fail', (err) => {
           expect(err.message).to.include('variable is not defined')
           expect(err.message).to.include(`Variables must either be defined within the \`cy.origin()\` command or passed in using the args option.`)
-          expect(err.stack).to.include(`Variables must either be defined within the \`cy.origin()\` command or passed in using the args option.`)
```

Comment:
Do not fear! The fact that this string is not in the stack does not mean it has been lost. It is in the message. There are no changes to this error message in the GUI or runner, aside from the corrected stack trace line number/columns. To test this, set up a `cy.origin` test that will fail, and simply compare the error output in the GUI and runner between this branch and `develop`